applications.yaml

```
@@ -310,6 +316,12 @@
   float_identifiers:
   - kind: Exe
     id: GoogleDriveFS.exe
+- name: Honeyview
+  identifier:
+    kind: Exe
+    id: Honeyview.exe
+  options:
+  - force
```

```suggestion
  identifier:
    kind: Class
    id: HoneyviewClassX
  options:
  - border_overflow
```

Honeyview seems to be managed fine to me, however it has a border overflow problem.